prime-router/src/testIntegration/resources/settings/organizations.yml

```
@@ -575,4 +575,116 @@
       enrichmentSchemaNames:
         - "classpath:/metadata/fhir_transforms/common/suppress-non-npi.yml,classpath:/metadata/fhir_transforms/common/suppress-non-npi.yml"
       timeZone: null
-      dateTimeFormat: "OFFSET"
\ No newline at end of file
+      dateTimeFormat: "OFFSET"
+- name: "mo-phd"
+  description: "Missouri Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "MO"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "mo-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/MO/MO-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MOELR^2.16.840.1.114222.4.3.3.36^ISO"
+        receivingApplicationOID: "2.16.840.1.114222.4.3.3.36"
+        receivingFacilityName: "MODHSS^2.16.840.1.114222.4.3.2.2.1.163^ISO"
+        receivingFacilityOID: "2.16.840.1.114222.4.3.2.2.1.163"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: null
+        reportingFacilityName: "CDC PRIME"
+        reportingFacilityId: "11D2030855"
+        reportingFacilityIdType: null
+        suppressQstForAoe: true
+        suppressHl7Fields: "MSH-19-3, PID-15-3, ORC-14-1, OBR-17-1"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: true
+        truncateHl7Fields: "MSH-3-1, PID-3-3, PID-3-6, ORC-2-2, ORC-7-10-5, OBR-2-2, OBR-3-2, SPM-2-1-2, SPM-2-2-2, ORC-21-1, OBX-23-1, BX-15.1, OBX-23.10 "
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: true
+          truncateHl7Fields:
+            - "MSH-3-1"
+            - "PID-3-3"
+            - "PID-3-6"
+            - "ORC-2-2"
+            - "ORC-7-10-5"
+            - "OBR-2-2"
+            - "OBR-3-2"
+            - "SPM-2-1-2"
+            - "SPM-2-2-2"
+            - "ORC-21-1"
+            - "OBX-23-1"
+            - "OBX-15.1"
+            - "OBX-23.10"
```

Haven't looked into why we have two settings for the truncation fields but we should keep it consistent just in case.
```suggestion
          truncateHl7Fields:
            - "MSH-3-1"
            - "PID-3-3"
            - "PID-3-6-1"
            - "ORC-2-2"
            - "ORC-7-10-5"
            - "OBR-2-2"
            - "OBR-3-2"
            - "SPM-2-1-2"
            - "SPM-2-2-2"
            - "ORC-21-1"
            - "OBX-23-1"
            - "OBX-15-1"
            - "OBX-23-10"
```